_saas-integrations/ga360/v1/ga360-v1.md

```diff
@@ -0,0 +1,163 @@
+
+api-type: "platform.ga360"
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Enterprise"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**A Google BigQuery and {{ integration.display_name }} account**. You need to have your BigQuery account configured to export to {{ integration.display_name }}. To learn how to configure this export, use [Google's step-by-step instructions](https://support.google.com/analytics/answer/3416092?hl=en&ref_topic=3416089)."
+
+setup-steps:
+  - title: "Locate your Project ID and Dataset ID"
+    anchor: "retrieve-project-dataset-ids"
+    content: |
+      1. Login to Google and go to your Google Cloud Console.
+      2. To locate your list of projects, click on the navigation menu in the upper left-hand corner of the page.
+      3. Hover over **IAM & Admin** to pop open its menu. Select **Manage Resources**.
+      4. On this page, navigate to the project that includes the dataset you want to replicate data from and click on it.
+      5. Your **Project ID** and all of your **Dataset IDs** will be visible. Make note of the Project ID and Dataset ID that you want to replicate and keep it readily available for the {{ integration.display_name }} integration configuration page.
+
+  - title: "Create a GCP IAM service account"
+    anchor: "create-gcp-iam-service-account"
+    content: |
+      {% for substep in step.substeps %}
+      - [Step 2.{{ forloop.index }}: {{ substep.title }}](#{{ substep.anchor }})
+      {% endfor %}
+
+    substeps:
+      - title: "Define the service account details"
+        anchor: "define-service-account-details"
+        content: |
+          {% include destinations/google-bigquery/create-iam-service-account.html type="define-service-account-details" %}
+
+      - title: "Assign BigQuery Admin permissions"
+        anchor: "assign-bigquery-admin-permissions"
+        content: |
+          {% include destinations/google-bigquery/create-iam-service-account.html type="assign-bq-admin" %}
```

Comment:
I'm not sure we need admin permissions in this service account since we're only running selects (vs. loader which has to create datasets, etc.) but I don't know what kinds of permissions this was tested with.